demos/button.html

```diff
@@ -105,9 +105,9 @@
           <label for="toggle-disabled" id="toggle-disabled-label">Disable buttons (excluding links)</label>
         </div>
 
-        <h1 class="mdc-typography--display2">Ripple Enabled</h1>
+        <h1 class="mdc-typography--headline3">Ripple Enabled</h1>
```

Comment:
I'm immediately going to ask this question because other people are going to ask it...

It seems weird that we're using headline3 for h1 and headline6 for something that isn't a headline element at all. Does design intentionally not have a direct 1:1 mapping between h1-h6 and the heading1-heading6 typography classes? And if not, why did they now dictate 6 typography scale names that just so happen to directly coincide with 6 HTML elements which suggest hierarchy?

Reply:
Design doesn't indicate what we should be putting the typography elements on. Are you advocating for putting the `headline` classes on their respective `h` tags?

Reply:
I mean... I'm wondering why design picked typography style names that bear a strong resemblance to native HTML elements (in terms of both name and number of them), if they're not intended to be directly related with those native HTML elements (and perhaps not intended to be directly tied to a hierarchy, whereas those tags generally exist for that purpose).

Again, I asked the question because I expect we're going to get the question a lot (see also #2240, which we got even before the new style names were introduced here).